test_junit_xml.py

```diff
@@ -73,8 +73,8 @@ def test_single_suite_no_test_cases(self):
 
         (ts, tcs) = serialize_and_read(
             TestSuite(
-                'test',
-                [],
+                name='test',
```

Comment:
Here, I just add attributes explicitly in constructor signature. I think I don't break anyone's code :___(